frontend/ui-picasso/app/stores/defi/polkadot/bonds/slice.ts

```diff
@@ -0,0 +1,58 @@
+import { BondOffer } from "@/stores/defi/polkadot/bonds/types";
```

Comment:
would prefer zustand store

Reply:
Once the other PR is ready, we can refactor this.